examples/python/interceptors/default_value/.gitignore

```diff
@@ -0,0 +1 @@
+*.pyc
```

Comment:
Why is this necessary? Doesn't the repository top-level `.gitignore` file also affect this subdirectory?